### ISSUE TITLE
tikv crash during volumerestore

### ISSUE THREAD

**Issue by zhongmin-amin** [tikv/tikv#16673]
### Steps to reproduce
1. do volumebackup
2. do volumerestore

### What did you expect?
1. volumerestore success

### What did happened?
1. tikv crash
2. 
![img_v3_0294_2c478296-9a3a-4cab-a931-e1a9e272e99g](https://github.com/tikv/tikv/assets/81375082/da122691-9262-48b0-807e-ab9126529937)

**Comment by zhangjinpeng87** [comment]
@BornChanger Is this issues fixed by https://github.com/tikv/tikv/pull/16706?

**Comment by BornChanger** [comment]
> @BornChanger Is this issues fixed by #16706?

No. It's a different issue and we don't have clue on it at present since the cluster has been cleaned up.

**Comment by BornChanger** [comment]
@jebter @zhongmin-amin since the problem is exposed one at the wal corruption analysis time and it is not re-produceable, the analysis stuck now, can we close it temporarily?

**Comment by jebter** [comment]
If it happens again, we will reopen it
